{
  "id": "EUVD-2021-0167",
  "enisaUuid": "b21cc0a5-6aad-3fea-954d-b00c48a9f40a",
  "description": "An issue was discovered in Pillow before 8.2.0. There is an out-of-bounds read in J2kDecode, in j2ku_graya_la.",
  "datePublished": "Jun 2, 2021, 4:15:00 PM",
  "dateUpdated": "Aug 27, 2021, 3:22:10 AM",
  "baseScore": 9.1,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
  "references": [
    "https://pillow.readthedocs.io/en/stable/releasenotes/8.2.0.html#cve-2021-25287-cve-2021-25288-fix-oob-read-in-jpeg2kdecode",
    "https://github.com/python-pillow/Pillow/pull/5377#issuecomment-833821470",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MQHA5HAIBOYI3R6HDWCLAGFTIQP767FL/",
    "https://github.com/advisories/GHSA-77gc-v2xv-rvvh",
    "https://nvd.nist.gov/vuln/detail/CVE-2021-25287",
    "https://github.com/python-pillow/Pillow/pull/5377/commits/3bf5eddb89afdf690eceaa52bc4d3546ba9a5f87",
    "https://github.com/python-pillow/Pillow/commit/3bf5eddb89afdf690eceaa52bc4d3546ba9a5f87",
    "https://github.com/pypa/advisory-database/tree/main/vulns/pillow/PYSEC-2021-137.yaml",
    "https://security.gentoo.org/glsa/202107-33",
    "https://www.suse.com/security/cve/CVE-2021-25287.html",
    "https://access.redhat.com/errata/RHSA-2021:4149",
    "https://ubuntu.com/security/CVE-2021-25287",
    "https://advisories.mageia.org/CVE-2021-25287.html",
    "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/MQHA5HAIBOYI3R6HDWCLAGFTIQP767FL/"
  ],
  "aliases": [
    "CVE-2021-25287",
    "GHSA-77gc-v2xv-rvvh",
    "PYSEC-2021-137"
  ],
  "assigner": "mitre",
  "epss": 0.34,
  "enisaIdProduct": [
    {
      "id": "cfcd5e72-bc22-396e-9299-63559e9fcc09",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "1adbdbfa-822b-335e-b238-1e49576ecc93",
      "vendor": {
        "name": "n/a"
      }
    }
  ],
  "cve": "CVE-2021-25287"
}